hail/src/main/scala/is/hail/cxx/SparkUtils.scala

```diff
@@ -10,17 +10,21 @@ import is.hail.utils.using
 
 import scala.collection.mutable
 
-class SparkUtils(mods: Array[(String, NativeModule)]) {
+class SparkUtils(mods: Array[(String, (Array[Byte], NativeModule))]) {
 
-  private[this] val loadedModules: Map[String, NativeModule] = mods.toMap
-  def getModule(id: String): (String, Array[Byte]) =
-    (loadedModules(id).getKey, loadedModules(id).getBinary)
+  type Literals = Array[Byte]
```

Comment:
This is kind of confusing since I think earlier a single `Literal` was represented by a byte array. This goes with my previous comment on type-naming.

Reply:
I've tried to clarify what's going on a little bit in the naming, but in essence:
- When we emit IRs, we create a placeholder variable for all the literals that we encounter. This is so we can emit literals as we see them, but we can ultimately bundle up all the literals for one function/module and ship them at the same time.
- After we generate the triplet for an IR, all the literals that go into one function get combined into a single tuple and serialized together (so the Array[Byte] that gets passed around is the entire (serialized) set of literals).
- When the literals are unserialized inside the function body, the individual literals (at this point, elements of the `literals` tuple) get extracted and bound to the variables that we used as placeholders.

Is there anything else that you think would help make this clearer in the code? Open to suggestions.

Reply:
I think the clarifications help. If I'm understanding correctly, the `Array[Byte]` in `EncodedLiterals` would be the same as `Literals`? Is it possible to use that type name instead of `Array[Byte]`? Then someone could track the type through and see the connection.

Reply:
yep! I'll do that.

Reply:
(fixed)